experiments/golden-results/StratoX-summary.txt

```diff
@@ -28,17 +28,27 @@ Calling function: Process_Pragma_Declaration
 Error message: Unsupported pragma: No strict aliasing
 Nkind: N_Pragma
 --
-Occurs: 30 times
+Occurs: 31 times
```

Comment:
I presume this is another "things continue further so they hit later errors"?

If this is what is going on, shouldn't the total number of errors stay the same?  Does that happen?